crates/proc_macro_api/src/rpc.rs

```diff
@@ -51,6 +51,9 @@ pub struct ExpansionTask {
     pub attributes: Option<Subtree>,
 
     pub lib: PathBuf,
+
+    /// Environment variables to set during macro expansion.
+    pub env: Vec<(String, String)>,
```

Comment:
Would `OsString` be a better fit here semantic-wise?

In analysis, we need to use `String`s, because only that's compatible with `env!`. 

Here, the consumre is external process, so OsString might be better?

But mostly, this is irrelevant.

Reply:
Yeah, that's true. But they come from `Env`, so we won't see non-UTF-8 variables.

Reply:
Yeah, in similar cases I generally prefer to shape the interface to the shape of the external world, rather than to the shape of an internal impl. 

It's less confusing to have some explicit glue code, than for two things share identical inteface by accident.